package.json

```diff
@@ -38,9 +39,9 @@
     "xstate:typegen": "pnpm -r xstate:typegen"
   },
   "dependencies": {
-    "@fuel-ui/config": "0.15.0-dev-c6f1a1b",
-    "@fuel-ui/css": "0.15.0-dev-c6f1a1b",
-    "@fuel-ui/react": "0.15.0-dev-c6f1a1b"
+    "@fuel-ui/config": "0.15.0-dev-9c7ba65",
```

Comment:
can the fuel-ui deps be pinned to a non-dev version yet?  I guess that should be done in a separate pr